custom/code_types.inc.php

```diff
@@ -697,7 +697,7 @@ function lookup_code_descriptions($codes, $desc_detail = "code_text")
                 continue;
             }
 
-            list($codetype, $code) = explode(':', $codestring);
+            list($codetype, $code, $modifier) = explode(':', $codestring);
```

Comment:
would document this exception on line 679 (and that will only work with internal codesets)